src/game_api/screen.hpp

```diff
@@ -226,79 +312,90 @@ class ScreenOptions : public Screen // ID: 5
     TextureRenderingInfo item_option_arrow_right;
     TextureRenderingInfo tooltip_background;
     TextureRenderingInfo progressbar_background; // brightness 'progressbar' background texture
-    TextureRenderingInfo unknown40;
-    TextureRenderingInfo progressbar_foreground;
-    TextureRenderingInfo progressbar_position_indicator;
-    TextureRenderingInfo sectionheader_background; // behind 'GRAPHICS' and 'AUDIO'
+    TextureRenderingInfo volume_progressbar_foreground;
+    TextureRenderingInfo progressbar_foreground; // the border
+    TextureRenderingInfo volume_progressbar_position_indicator;
+    TextureRenderingInfo sectionheader_background; // behind 'GRAPHICS' and 'AUDIO' the black bars
 
-    TextureRenderingInfo unknown44;
+    TextureRenderingInfo unknown44; // could be something else, or just not used in PC version or something
     TextureRenderingInfo unknown45;
-    float topleft_woodpanel_esc_slidein_timer;
-    float text_fadein_timer;
-    float vertical_scroll_effect_timer;
+    TextureRenderingInfo unknown46;
+    TextureRenderingInfo unknown47;
+    TextureRenderingInfo unknown48;
+    float unknown48a; // was supposto be topleft_woodpanel_esc_slidein?
+    float text_fadein;
+    float vertical_scroll_effect;
     uint8_t unknown49;
-    uint8_t unknown50;
-    uint8_t unknown51;
+    bool item_visiable;
+    bool item_highlight;
```

Comment:
item / option / record / element ? what's the best name for it? i want to standardize it since it comes often in few screens and with couple things related to it

Reply:
Well I have no idea what it is, but item sounds generic enough...

Reply:
I think the most common is "menu option" is what i meant